common/Makefile.am

```diff
@@ -3,10 +3,12 @@ libyami_common_source_c = \
 	utils.cpp \
 	nalreader.cpp \
 	surfacepool.cpp \
+	pooledframeallocator.cpp \
```

Comment:
Please use PooledFrameAllocator.cpp. We should follow webkit style https://webkit.org/code-style-guidelines/ for new file